platform/centec-arm64/sonic-platform-modules-e530/kdal/dal_kernel.c

```diff
@@ -0,0 +1,2234 @@
+/**
+ @file dal_kernal.c
```

Comment:
so many duplicated files. in e530, e582. Can you consolidate all those asic drivers into one. each time, it requires reviewer to review this file again and again.

Reply:
OK, code will be optimized, thank you.